test/Sema/diag_ambiguous_overloads.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -swift-version 4
```

Comment:
Sorry I didn't actually comment on this right away but we can't really switch version to 4 here, we'd have to add additional `test/Sema/diag_ambiguous_overloads_swift4.swift` and have it specify `-swift-version 4` there.

More interesting question is why `-swift-version 4` is even required here, which tests change?

Reply:
This is because SR-7295 compiles in Swift 3 while has error in Swift 4. Anyway, I shouldn't apply `-swift-version 4` for all test cases in that file. I created another file to have SR-7295, as you suggested.